packages/shared/src/components/modals/CreateMyFeedModal.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { ModalProps } from './StyledModal';
+import { ResponsiveModal } from './ResponsiveModal';
+import styles from './AccountDetailsModal.module.css';
+import TagsFilter from '../filters/TagsFilter';
+import CreateFeedFilterButton from '../CreateFeedFilterButton';
+import PlusIcon from '../../../icons/plus.svg';
+import XIcon from '../../../icons/x.svg';
+import { Button, ButtonSize } from '../buttons/Button';
+
+type TypeProps = {
+  type: string;
+};
+
+type CreateMyFeedModalProps = TypeProps & ModalProps;
+
+type LayoutModalProps = TypeProps & Partial<ModalProps>;
+
+type CloseButtonProps = {
+  size?: ButtonSize;
+  position?: string;
+  closeButtonClassName?: string;
+} & Partial<ModalProps>;
+
+const buttonClass = {
+  v2: 'small',
+  v3: 'medium',
+  v4: 'w-40',
+  v5: 'w-40 ml-3',
+};
+
+const headerTitle = {
+  v3: 'Choose tags to follow',
+  v4: 'Feed filters',
+  v5: 'Choose tags to follow',
+};
+
+const headerClass = {
+  v4: 'flex-row-reverse',
+};
+
+const getFooter = ({ type, onRequestClose }: LayoutModalProps) => {
```

Comment:
The difference should be only he name of the function?

Reply:
Would also be how we render it. Reading the render would allow us to easily understand we expect a component - a function can return anything, so we won't have to look at it if not really necessary.